src/networking.c

```diff
@@ -115,7 +115,6 @@ client *createClient(connection *conn) {
      * in the context of a client. When commands are executed in other
      * contexts (for instance a Lua script) we need a non connected client. */
     if (conn) {
-        connNonBlock(conn);
```

Comment:
Looks like this was never really needed because `connSocketConnect` already sets the socket to non blocking, and in other cases the `acceptXXXHandler` did so. **But** if I'm missing something then it means that removing this might cause a bug in cases `createClient` is called not after accept or `connSocketConnect`.

Reply:
as far as i could tell, createClient (with non-null conn) was only used in these two cases.